README.md

```diff
@@ -57,6 +57,37 @@ cd ./scripts/gateway
 $ make migrate-version
 ```
 
+## Gear DB migration
+
+**Add a version**
+
+```
+$ export GEAR=<gear_name>
+$ export REVISION=<revision_description>
+$ make -f scripts/gateway/Makefile add-version MIGRATION_DIR=cmd/migrate/gear/${GEAR} REVISION=${REVISION}
```

Comment:
It would be better if we have a `common.mk` in core, and gear has its own Makefile?